python/ray/tune/callback.py

```diff
@@ -343,3 +347,24 @@ def on_checkpoint(self, **info):
     def on_experiment_end(self, **info):
         for callback in self._callbacks:
             callback.on_experiment_end(**info)
+
+    def get_state(self) -> Optional[Dict]:
+        state = {}
+        any_stateful_callbacks = False
+        for i, callback in enumerate(self._callbacks):
+            callback_state = None
+            try:
+                callback_state = callback.get_state()
+                any_stateful_callbacks = True
+            except NotImplementedError:
```

Comment:
I have replaced the `get_state` with a default of returning None (stateless). No need to check for this error anymore. I think an attribute error should get raised immediately instead of caught.